build.gradle

```diff
@@ -161,8 +161,8 @@ subprojects {
     maxHeapSize = "1g"
     useJUnitPlatform()
     testlogger {
-      showSummary true
```

Comment:
why are you changing these?

Reply:
it was Ricsi's change, so I guess this PR might not be on top of the HEAD

Reply:
Let me rebase, when I moved the PR to CB-2.55.0 branch it picked up this commit that I haven't noticed.